pkg/sql/sem/tree/create_logical_replication.go

```diff
@@ -163,6 +168,14 @@ func (o *LogicalReplicationOptions) CombineWith(other *LogicalReplicationOptions
 		}
 	}
 
+	if o.FilterRangefeed != nil {
+		if other.FilterRangefeed != nil {
+			return errors.New("IGNORE_TTL_DELETES option specified multiple times")
```

Comment:
drive-by comment: should this be IGNORE_CDC_IGNORED_TTL_DELETES?

Reply:
Good catch, will fix in another PR